src/test/java/no/unit/nva/model/instancetypes/journal/JournalArticleTest.java

```diff
@@ -18,14 +27,79 @@ void journalArticleReturnsObjectWhenPeerReviewIsTrueOrFalse(boolean peerReviewed
         assertDoesNotThrow(() -> generateJournalArticle(testData));
     }
 
+    @Test
+    public void canSerializeAndDeserializeJournalArticleWithContentTypeResearchArticle()
+            throws JsonProcessingException {
+
+        JournalArticle expectedJournalArticle = new JournalArticle.Builder()
+                .withContent(JournalArticleContentType.PROFESSIONAL_ARTICLE).build();
+        String expectedJson = objectMapper.writeValueAsString(expectedJournalArticle);
+
+        JournalArticle actualJournalArticle = objectMapper.readValue(expectedJson, JournalArticle.class);
+        assertEquals(expectedJournalArticle, actualJournalArticle);
+
+        String actualJson = objectMapper.writeValueAsString(expectedJournalArticle);
+        assertEquals(expectedJson, actualJson);
+    }
+
+
+    @DisplayName("Test JournalArticle with content can be serialized/deserialized")
+    @ParameterizedTest(name = "Test JournalArticle with Content type {0} can be (de-)serialized")
+    @ValueSource(strings = {
+            "Research article",
+            "Review article",
+            "Case report",
+            "Study protocol",
+            "Professional article",
+            "Popular science article"
+    })
+    void publicationReturnsJsonWhenInputIsValid(String content) throws IOException {
+
+        JournalTestData expectedJournalArticleTestData = new JournalTestData();
+        expectedJournalArticleTestData.setContent(content);
+
+        JournalArticle expectedJournalArticle = generateJournalArticle(expectedJournalArticleTestData);
+        String expectedJson = objectMapper.writeValueAsString(expectedJournalArticle);
+
+        JournalArticle actualJournalArticle = objectMapper.readValue(expectedJson, JournalArticle.class);
+        assertEquals(expectedJournalArticle, actualJournalArticle);
+
+        String actualJson = objectMapper.writeValueAsString(expectedJournalArticle);
+        assertEquals(expectedJson, actualJson);
+    }
+
+    @Test
+    public void canSerializeJournalArticleWithContentTypeWithoutDataloss() throws JsonProcessingException {
+        String contentTypeString = "Research article";
```

Comment:
Use the enum and the getValue() method: `JournalArticleContentType.RESEARCH_ARTICLE.getValue();`